### PR TITLE
Add Fedora 37 nightly builds

### PR DESCRIPTION
**Pull request by junkblocker**

### CODE REVIEW

File: ci/generate-workflows.py [hunk 1]
```diff
@@ -833,10 +833,10 @@ def tag(self):
     Target(name="centos9", container="quay.io/centos/centos:stream9"),
     Target(name="macos", os="macos-11"),
     # https://fedoraproject.org/wiki/End_of_life?rd=LifeCycle/EOL
-    Target(container="fedora:33"),
```
Comment (wez) [inline ci/generate-workflows.py:836]: Please also `git rm` the `gen_fedora33*` workflow files and add that to this PR!

File: docs/install/linux.markdown [hunk 2]
```diff
@@ -114,6 +114,7 @@ on most Linux systems.
 |Fedora34    |[{{ fedora34_rpm_stable_asset }}]({{ fedora34_rpm_stable }}) |[{{ fedora34_rpm_nightly_asset }}]({{ fedora34_rpm_nightly }})|
 |Fedora35    |[{{ fedora35_rpm_stable_asset }}]({{ fedora35_rpm_stable }}) |[{{ fedora35_rpm_nightly_asset }}]({{ fedora35_rpm_nightly }})|
 |Fedora36    |[{{ fedora36_rpm_stable_asset }}]({{ fedora36_rpm_stable }})|[{{ fedora36_rpm_nightly_asset }}]({{ fedora36_rpm_nightly }})|
+|Fedora37    |Nightly builds only |[{{ fedora37_rpm_nightly_asset }}]({{ fedora37_rpm_nightly }})|
```
Comment (wez) [inline docs/install/linux.markdown:117]: and edit the 33 entry above to replace the nightly download cell with "No longer provided" or something like that

File: .github/workflows/gen_fedora37_tag.yml [hunk 3]
```diff
@@ -84,10 +86,7 @@ jobs:
       - name: "Download artifact"
         uses: actions/download-artifact@v3
         with:
-          name: "fedora33"
-      - name: "Checksum"
```
Comment (proski) [inline .github/workflows/gen_fedora37_tag.yml:88]: Why is the Checksum step removed? Can it be that `gen_fedora37_continuous.yml` was copied from `gen_fedora36_continuous.yml` prior to #2114 ?

File: docs/install/linux.markdown [hunk 4]
```diff
@@ -110,10 +110,11 @@ on most Linux systems.
 |CentOS7     |[{{ centos7_rpm_stable_asset }}]({{ centos7_rpm_stable }}) |[{{ centos7_rpm_nightly_asset }}]({{ centos7_rpm_nightly }})|
 |CentOS8     |[{{ centos8_rpm_stable_asset }}]({{ centos8_rpm_stable }}) |[{{ centos8_rpm_nightly_asset }}]({{ centos8_rpm_nightly }})|
 |CentOS9     |[{{ centos9_rpm_stable_asset }}]({{ centos9_rpm_stable }})|[{{ centos9_rpm_nightly_asset }}]({{ centos9_rpm_nightly }})|
-|Fedora33    |[{{ fedora33_rpm_stable_asset }}]({{ fedora33_rpm_stable }}) |[{{ fedora33_rpm_nightly_asset }}]({{ fedora33_rpm_nightly }})|
+|Fedora33    |[{{ fedora33_rpm_stable_asset }}]({{ fedora33_rpm_stable }}) |No longer provided|
```
Comment (proski) [inline docs/install/linux.markdown:113]: I would just remove that line. It's quite obvious that Fedora 33 is unsupported. In fact, Fedora 34 and 35 are also unsupported.

**Review by proski — COMMENTED** [review]
I've checked the files against those for Fedora 35 and 36. The changes look reasonable except as noted.

File: .github/workflows/gen_fedora37_tag.yml (hunk 3 above)
Comment (junkblocker) [reply .github/workflows/gen_fedora37_tag.yml:88]: Good catch. Fixed.

File: docs/install/linux.markdown (hunk 4 above)
Comment (junkblocker) [reply docs/install/linux.markdown:113]: Kept it as it is for now as suggested by wez.

**Comment by wez** [comment]
Thanks!

**Pull request merged**